components/bh1750/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 idf_component_register(
     SRCS "bh1750.c"
     INCLUDE_DIRS "include"
+    # Requires driver for I2C header file
+    PRIV_REQUIRES "driver"
```

Comment:
We include `driver/i2c.h` from public header files (e.g. [bh1750](https://github.com/espressif/esp-bsp/blob/master/components/bh1750/include/bh1750.h#L13))

Shouldn't this be public `REQUIRES "driver"` too?

Reply:
I wasn't sure, I will set it as public then

Reply:
@tore-espressif Updated!

Reply:
Thanks! I will merge after CI passes